cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/elements/EnergyConsumerConversion.java

```diff
@@ -40,32 +40,46 @@ public void convert() {
     }
 
     private static void setLoadDetail(String type, Load load) {
-        if (type.endsWith("#ConformLoad")) { // ConformLoad represent loads that follow a daily load change pattern where the pattern can be used to scale the load with a system load
-            load.newExtension(LoadDetailAdder.class)
-                    .withFixedActivePower(0)
-                    .withFixedReactivePower(0)
-                    .withVariableActivePower((float) load.getP0())
-                    .withVariableReactivePower((float) load.getQ0())
-                    .add();
-        } else if (type.endsWith("#NonConformLoad")) { // does not participate in scaling
-            load.newExtension(LoadDetailAdder.class)
-                    .withFixedActivePower((float) load.getP0())
-                    .withFixedReactivePower((float) load.getQ0())
-                    .withVariableActivePower(0)
-                    .withVariableReactivePower(0)
-                    .add();
+        float p0 = (float) load.getP0();
+        float q0 = (float) load.getQ0();
+        if (!Float.isNaN(p0) && !Float.isNaN(q0)) {
+            if (type.endsWith("#ConformLoad")) { // ConformLoad represent loads that follow a daily load change pattern where the pattern can be used to scale the load with a system load
+                load.newExtension(LoadDetailAdder.class)
+                        .withFixedActivePower(0)
+                        .withFixedReactivePower(0)
+                        .withVariableActivePower(p0)
+                        .withVariableReactivePower(q0)
+                        .add();
+            } else if (type.endsWith("#NonConformLoad")) { // does not participate in scaling
+                load.newExtension(LoadDetailAdder.class)
+                        .withFixedActivePower(p0)
+                        .withFixedReactivePower(q0)
+                        .withVariableActivePower(0)
+                        .withVariableReactivePower(0)
+                        .add();
+            }
+            // else: EnergyConsumer - undefined
         }
-        // else: EnergyConsumer - undefined
     }
 
     @Override
     protected double p0() {
-        return powerFlow().defined() ? powerFlow().p() : p.asDouble("pFixed", 0.0);
+        return powerFlow().defined() ? powerFlow().p() : p.asDouble("pFixed", () -> {
+            if (context.cgmes().hasOnlyEquipmentProfile()) {
+                return Double.NaN;
+            }
+            return 0.0;
```

Comment:
What is the aim of this `0.0` value ?

Reply:
It is for default values when there is several profiles (not only EQ and boundaries) in the CGMES file

Reply:
Your proposal is the cleanest one for sure, but not maybe the most realistic: many IGM will have a validation level at EQ, whereas we really want to run a load flow on it after import. I can imagine that in the TYNDP files, we are far from a SSH validation level... what do you think ?